packages/polaris-test/tests/online-pagination.test.ts

```diff
@@ -0,0 +1,109 @@
+import { createServers } from '../test-utils/tests-servers-util';
+import { polarisTest } from '../test-utils/polaris-test';
+import { graphqlRawRequest, graphQLRequest } from '../test-utils/graphql-client';
+import * as createBook from './jsonRequestsAndHeaders/createBook.json';
+import * as onlineBooksPagination from './jsonRequestsAndHeaders/onlineBooksPagination.json';
+
+const setUp = async (): Promise<any[]> => {
+    const bookIds: any[] = [];
+    for (let i = 1; i <= 10; i++) {
+        const res = await graphQLRequest(createBook.request, {}, { title: `book${i}` });
+        bookIds.push(res.createBook.id);
+    }
+    return bookIds;
```

Comment:
you don't really use the bookIds, why return them and why do tear down? every test you have a new server up

Reply:
you are correct, i forgot it there.. good comment